Client.xcodeproj/project.pbxproj

```diff
@@ -5251,7 +5250,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "/usr/local/bin/carthage copy-frameworks";
+			shellScript = "/usr/local/bin/carthage copy-frameworks\n";
```

Comment:
This is done by Xcode-11 automatically.. If I remove the new line, the next time I go to build settings.. it comes back..